.github/workflows/test.yml

```
@@ -0,0 +1,76 @@
+name: Test
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron: |
+        0 0 * * 0
+
+jobs:
+  test:
+    name: "Ruby ${{ matrix.ruby-version }}: ${{ matrix.runs-on }}"
+    strategy:
+      # To avoid high frequency datasets parquet download in a short time.
+      max-parallel: 1
+      fail-fast: false
+      matrix:
+        ruby-version:
+          - "2.7"
+          - "3.0"
+          - "3.1"
+        runs-on:
+          - macos-latest
+          - ubuntu-latest
+          - windows-latest
+    runs-on: ${{ matrix.runs-on }}
+    env:
+      # We can invalidate the current cache by updating this.
+      CACHE_VERSION: "2022-08-27"
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby-version }}
+      - uses: actions/cache@v3
+        if: |
+          runner.os == 'Linux'
+        with:
+          path: |
+            ~/.cache/red-datasets-parquet
```

We can use the same directory as Red Datasets because we use the same cache directory as Red Datasets:

```suggestion
            ~/.cache/red-datasets
```

---

fix: [7fbc02e](https://github.com/red-data-tools/red-datasets-parquet/pull/3/commits/7fbc02e5b0cf5e76ba36f0cc27abed9fd84e87ad)